recipe/meta.yaml

```diff
@@ -23,10 +23,18 @@ requirements:
   run:
     - python
     - numpy
+  run_constrained:
+    # From the readme
+    # pyqtgraph has had some incompatibilities with PySide2 versions 5.6-5.11, and we recommend you avoid those versions if possible
+    - pyside2 >=5.12
     # pyqtgraph works with pyqt4 or pyqt5. Don't pin it here.
-    - pyqt
+    # - pyqt
```

Comment:
```suggestion
     - pyqt
```

Reply:
it doesn't really constrain it does it :)

Reply:
You are right! :)
I just would like to have it there as you are adding pyside there, to remember that the user can chose between those two
:D

Reply:
yep yep. That is why i had left it there in a comment. Thanks for understanding the need for both!